README.md

```diff
@@ -15,9 +15,17 @@ You can see all "ddev" usages using the help commands, like `ddev -h`, `ddev sta
 * *Currently only tested on Ubuntu, please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue on another flavor.*
 
 ## Installation
+### Installation Script
+You can paste this line of code to your terminal to download, verify, and install ddev using our install script:
```

Comment:
How about "install" being a link to https://github.com/drud/ddev/blob/master/install_ddev.sh (the actual github non-raw page), for those who don't want to run magic unread scripts?